ciftools/binary/data.py

```diff
@@ -0,0 +1,184 @@
+from typing import Any, Dict, List, Optional, Union
+
+import numpy as np
+from ciftools.binary.decoder import decode_cif_data
+from ciftools.binary.encoded_data import EncodedCIFCategory, EncodedCIFColumn, EncodedCIFFile
+from ciftools.models.data import CIFCategory, CIFColumn, CIFDataBlock, CIFFile, CIFValuePresenceEnum
+
+
+class BinaryCIFColumn(CIFColumn):
+    def __init__(
+        self,
+        name: str,
+        values: np.ndarray,
+        value_mask: Union[np.ndarray, None],
```

Comment:
Name `value_mask` is a bit misleading here as it's not a classic mask (array of bool)